cranelift/filetests/filetests/isa/x64/sqmul_round_sat.clif

```diff
@@ -13,7 +13,7 @@ block0(v0: i16x8, v1: i16x8):
 ; block0:
 ;   pmulhrsw %xmm1, %xmm0
 ;   movdqa %xmm0, %xmm5
-;   pcmpeqw %xmm5, const(0), %xmm5
```

Comment:
@abrown This filetest update bothers me since the original result that includes "const(0)" is not x86 syntax. Because it is not x86 syntax I am not sure how I can directly map this previous result to the current change to assume the blessed update is correct.